### PR TITLE
doc: update quick start

### PR DESCRIPTION
**Pull request by xiang90**
**1.Describe what this PR did**

clean up the first section of quick start

### CODE REVIEW

File: INSTALLATION.md [hunk 1]
```diff
@@ -1,8 +1,10 @@
-# Installation
+# Quick-Start
 
-We have two different ways to experience Pouch, one for end-users, the other for developers. In the former one, installation can help you with Pouch installing out of box. The latter one is for those who wish to get involved in Pouch with hacking.
+Two quick-start are provided, one for end-users, the other for developers.
```
Comment (allencloud) [inline INSTALLATION.md]: Do we need to use `Two quick starts` instead of `Two quick start`? Just double check.

Comment (xiang90) [reply INSTALLATION.md]: will fix.

**Comment by xiang90** [comment]
@allencloud updated

**Comment by allencloud** [comment]
LGTM, merging...

**Pull request merged**